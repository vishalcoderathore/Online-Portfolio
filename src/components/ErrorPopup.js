import React from 'react';

class ErrorPopup extends React.Component {
    render() {
        return (
            <div className="form__error">
                <h5>
                    Error
                </h5>
                <p>First Name, Last Name and Email are required fields</p>
            </div>
        )
    }
}

export default ErrorPopup;