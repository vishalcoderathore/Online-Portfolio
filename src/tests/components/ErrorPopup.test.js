import React from 'react';
import { shallow } from 'enzyme';
import ErrorPopup from '../../components/ErrorPopup';

test('should render Loading Page correctly', () => {
    const wrapper = shallow(<ErrorPopup />);
    expect(wrapper).toMatchSnapshot();
});