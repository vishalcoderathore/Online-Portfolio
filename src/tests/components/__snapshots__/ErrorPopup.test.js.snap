// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Loading Page correctly 1`] = `
<div
  className="form__error"
>
  <h5>
    Error
  </h5>
  <p>
    First Name, Last Name and Email are required fields
  </p>
</div>
`;
